Makefile

```diff
@@ -0,0 +1,3 @@
+.PHONY test/e2e:
+test/e2e:
+	sh scripts/e2e.sh
```

Comment:
Should set the shell to bash see https://github.com/integr8ly/heimdall/blob/master/Makefile#L8, and remove sh here